src/NCronJob/Execution/JobExecutor.cs

```diff
@@ -154,8 +154,8 @@ public void InformDependentJobs(JobExecutionContext context, bool success)
 
         var jobRun = context.JobRun;
         var dependencies = success
-            ? jobRegistry.GetDependentSuccessJobTypes(jobRun.JobDefinition.Type)
-            : jobRegistry.GetDependentFaultedJobTypes(jobRun.JobDefinition.Type);
+            ? jobRegistry.GetDependentSuccessJobTypes(jobRun.JobDefinition)
```

Comment:
Would it make sense to remove the jobRegistry's `GetDependentSuccessJobTypes` and hold this information directly on the jobdefinition or would it be to convoluted?

Reply:
I don't know yet. I'll work on this and `DepedentJobRegistryEntry` in a follow up PR.